packages/repository/src/common-types.ts

```diff
@@ -51,7 +51,7 @@ export type DataObject<T> = T | AnyObject;
 /**
  * Type alias for Node.js options object
  */
-export type Options = AnyObject | null | undefined;
+export type Options = AnyObject | undefined;
```

Comment:
Why is `null` no longer needed here?

Reply:
See https://basarat.gitbooks.io/typescript/docs/javascript/null-undefined.html

Reply:
We should even remove `undefined` from `Options`. It should be `options?: Options` instead.